lib/state/appActions.js

```diff
@@ -257,6 +258,13 @@ appActions.login = function(credentials, options, cb) {
       return cb(err);
     }
     self._logMetric(self.trackedState.LOGIN_SUCCESS);
+    
+    var acks = localStore.getItem('acknowledgments');
```

Comment:
OK, I get that - re: the nested object and re-use. If you could move to `_afterLoginPage` along with `devices` that'd be great, just to keep things together.